mobile/lib/presentation/widgets/bottom_sheet/map_bottom_sheet.widget.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:immich_mobile/presentation/widgets/bottom_sheet/base_bottom_sheet.widget.dart';
+import 'package:immich_mobile/presentation/widgets/map/map.state.dart';
+import 'package:immich_mobile/presentation/widgets/timeline/timeline.widget.dart';
+import 'package:immich_mobile/providers/infrastructure/timeline.provider.dart';
+import 'package:maplibre_gl/maplibre_gl.dart';
+
+class MapBottomSheet extends ConsumerWidget {
+  const MapBottomSheet({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    LatLngBounds bounds = ref.watch(mapStateProvider.select((s) => s.bounds));
+
+    ref.listen(mapStateProvider, (previous, next) async {
+      bounds = next.bounds;
+    });
+
+    return ProviderScope(
+      overrides: [
+        // TODO: refactor (timeline): when ProviderScope changed, we should refresh timeline
+        timelineServiceProvider.overrideWith((ref) {
+          final timelineService =
+              ref.watch(timelineFactoryProvider).map(bounds);
+          ref.onDispose(timelineService.dispose);
+          return timelineService;
+        }),
+      ],
+      child: const BaseBottomSheet(
+        initialChildSize: 0.25,
+        shouldCloseOnMinExtent: false,
+        actions: [],
+        slivers: [
+          SliverToBoxAdapter(
+            // TODO: Need refactor timeline and bottom sheet first
+            child: SizedBox(
+              height: 600,
+              width: 800,
+              child: Timeline(),
+            ),
+          ),
```

Comment:
Timeline is already a Sliver widget. Why'd we want to convert it to a Box widget then wrap it with a `SliverToboxAdapter`?

We can do this instead:
```suggestion
          SliverFillRemaining(
            child: Timeline(),
          ),
```